docs/manual/setup/getting-started.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Getting Started with Vector
+description: Getting Started with Vector, the "Hello World" of Vector tutorials.
+---
+
+Welcome to Vector! 🎈
+
+Vector is a high-performance observability data platform that allows you to collect, transform, and route all your
+logs and metrics.
+
+In this getting started guide, we'll walk you through using Vector for the first time. We'll install Vector
+and create our first observability data pipeline so you can begin to see what Vector can do.
+
+## Install Vector
+
+Installing Vector is quick and easy. We can use this handy installation script:
+
+```shell
+curl --proto '=https' --tlsv1.2 -sSf https://sh.vector.dev | sh
+```
+
+Or you can [choose your preferred installation method][docs.installation].
+
+Once we have Vector installed, let's check to make sure that it's working correctly:
+
+```shell
+vector --version
+```
+
+## Configure Vector
+
+Vector topologies are defined with a [configuration file][docs.setup.configuration] that tells it what
+[components][pages.components] to run and how they should interact. Vector topologies are made up of three
+types of components:
+
+* [**Sources**][docs.sources] collect or receive data from observability data sources into Vector
+* [**Transforms**][docs.transforms] can manipulate or change that observability data inside Vector
+* [**Sinks**][docs.sinks] send data onwards from Vector to external services or destinations
+
+Let's create a configuration file:
+
+```toml title="vector.toml"
+[sources.in]
+  type = "stdin"
+
+[sinks.out]
+  inputs = ["in"]
+  type = "console"
+  encoding.codec = "text"
+```
+
+Each component has a unique identifier and is prefixed with the type of component, for example, `sources` for a
+source. Our first component, `sources.in`, uses the [`stdin` source][docs.sources.stdin] which tells Vector to receive
+data over stdin and is identified as `in`.
+
+Our second component, `sinks.out`, uses the [`console` source][docs.sinks.console] which tells Vector to simply print
+the data to stdout. The `encoding.codec` option tells Vector to print the data as plain text (unencoded).
+The `inputs` option creates a pipeline from our source, which receives events, to our sink, which sends those
+events onwards. The `inputs` option in our `sinks.out` component tells Vector where this sink's events are coming
+from. In our case, events are received from our other component, the source identified as `in`.
```

Comment:
```suggestion
The `inputs` option in our `sinks.out` component tells Vector where this sink's events are coming
from. In our case, events are received from our other component, the source identified as `in`.
```

Reply:
I like calling out the pipeline concept? Can we do that another way?

Reply:
I think so, it just felt kind of word and repetitive since the following sentence saying something similar.